pom.xml

```diff
@@ -262,12 +262,13 @@
          -->
         <version.asciidoctor.plugin>2.2.2</version.asciidoctor.plugin>
         <version.org.jacoco>0.8.7</version.org.jacoco>
-        <version.org.jboss.galleon>5.0.3.Final</version.org.jboss.galleon>
+        <version.org.jboss.galleon>5.0.5.Final</version.org.jboss.galleon>
         <version.org.wildfly.build-tools>1.2.12.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.8.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.common>1.6.0.Final</version.org.wildfly.common>
-        <version.org.wildfly.galleon-plugins>6.0.0.Alpha6</version.org.wildfly.galleon-plugins>
+        <version.org.wildfly.galleon-plugins>6.0.0.Alpha7</version.org.wildfly.galleon-plugins>
         <version.org.wildfly.jar.plugin>7.0.0.Final</version.org.wildfly.jar.plugin>
```

Comment:
@spyrkob Please remove this line. Thanks!

Reply:
@bstansberry is the galleon-plugin upgraded in a separate PR? The upgrade is needed to include https://github.com/wildfly/galleon-plugins/pull/234

Reply:
@spyrkob It's upgraded in this PR. :) But you did it by duplicating the version property on L271 instead of changing the value on the existing L270. So L270 is now cruft.

Sorry; I should have put my comment after L271 so what I meant was clearer.

Reply:
@spyrkob are you able to address the comment from @bstansberry so this can be approved?

Reply:
@bstansberry 🤦 sorry I didn't notice that...

@darranl I pushed the change now, apologies for delay